deploy/crds.yaml

```diff
@@ -1,24 +1,27 @@
+---
```

Comment:
Is this some code gen error? We have an empty yaml doc then `v1beta1` CRD, shouldn't it be `v1`?

Reply:
Good catch this issue was caused by some extra new lines in the crd generation. Should be fixed now by removing them.